src/weather_gen.cpp

```diff
@@ -314,13 +318,13 @@ void weather_generator::test_weather() const
 weather_generator weather_generator::load( JsonObject &jo )
 {
     weather_generator ret;
-    ret.base_temperature = jo.get_float( "base_temperature", 6.5 );
-    ret.base_humidity = jo.get_float( "base_humidity", 66.0 );
-    ret.base_pressure = jo.get_float( "base_pressure", 1015.0 );
-    ret.base_acid = jo.get_float( "base_acid", 1015.0 );
-    ret.base_wind = jo.get_float( "base_wind", 5.7 );
-    ret.base_wind_distrib_peaks = jo.get_int( "base_wind_distrib_peaks", 30 );
-    ret.base_wind_season_variation = jo.get_int( "base_wind_season_variation", 64 );
+    ret.base_temperature = jo.get_float( "base_temperature", 0.0 );
+    ret.base_humidity = jo.get_float( "base_humidity", 50.0 );
+    ret.base_pressure = jo.get_float( "base_pressure", 0.0 );
+    ret.base_acid = jo.get_float( "base_acid", 0.0 );
+    ret.base_wind = jo.get_float( "base_wind", 0.0 );
```

Comment:
I looked over the data output, and I saw that windspeed average was significantly lower than usual, and had far less variability over the year.

Infact the highest windspeed for the entire year was 13mph

Would this be due to this change? and also above to peaks and variation, and also the change to `W = ` algorithm in get_weather() ? you didn't specify any rationale for change in windpower, although the changes to humidity pressure and precipitation look perfect.

Reply:
Yeah, I simply haven't looked at wind data or adjusted the wind code in this PR, so it relates to pressure the same way it used to. Should I overhaul wind, too?

Reply:
You set base_wind, base_wind_distrib_peaks, and base_wind_season_variation to 0, in this block of code. and changed the algorithm for W =  in get_weather().

Wind , as far as I can tell, dosnt need overhauling because it was set to new england data in the same way as you are doing for pressure and humidity. it could be that your changes to pressure are resulting in a flatter and lower wind distribution, due to the relationship that it previously had with the pressure value.

Could you output the data , but with no changes to those wind variables?

Reply:
Hmm, actually those are default loaded values if regional settings dont specify any. so it should still be the same, assuming you are using the standard regional settings.

It must be the pressure to wind relationship is different now.... may require some tweaking to get the previous wind behaviour with the new pressure

Reply:
That's right: the default region settings are just less redundantly specified now. And the change on the line with the assignment to `W` is just a variable rename. The pressure-to-wind relationship is the same, but pressure is different, so wind is different.

Reply:
It's up to you, wind is not vastly broken after this or anything, but it would be nice if you could find a relationship between wind and the new pressure that preserves as much of the old wind data distribution as possible - as that was matched to real-world data as close as possible in a previous PR.

If you want to do that in a separate PR, that would be fine, imo.

Reply:
Okay, let's focus on the other variables for now and maybe I'll do wind in a new PR.